src/components/SwapTokenDetails/SwapTokenDetails.tsx

```diff
@@ -67,9 +64,20 @@ const SwapTokenDetails: React.FC<{
         latestBlock,
       );
       setPriceData(tokenPriceData);
+
+      const [newPrice, oneDayPrice] = await getEthPrice();
+      const tokenInfo = await getTokenInfo(newPrice, oneDayPrice, tokenAddress);
+      if (tokenInfo) {
+        setTokenData(tokenInfo[0]);
```

Comment:
Try not to use indexers multiple times e.g 
let token0 = token info[0]

Reply:
Pushed a commit with update